dev-python/ipykernel/ipykernel-6.15.1.ebuild

```diff
@@ -27,7 +27,7 @@ RDEPEND="
 	dev-python/psutil[${PYTHON_USEDEP}]
 	>=dev-python/pyzmq-17[${PYTHON_USEDEP}]
 	>=dev-python/traitlets-5.1.0[${PYTHON_USEDEP}]
-	>=www-servers/tornado-6.1[${PYTHON_USEDEP}]
+	>=dev-python/tornado-6.1[${PYTHON_USEDEP}]
```

Comment:
Would be nice to keep things sorted where they seemed like being sorted originally.